src/features/newTab/default/clock/index.tsx

```diff
@@ -66,3 +68,5 @@ export class Clock extends React.PureComponent<{}, ClockState> {
     )
   }
 }
+
+export const ClockWidget = createWidget(Clock)
```

Comment:
Just a nit to say that this (and some other things in this PR) breaks backwards compatibility with the brave-core consumer, so any gap between this PR being merged with brave-ui's sha that brave-core is using, and the sister brave-core PR being merged, will result in broken builds. But the remedy for now is just to make sure they are merged very quickly together :-)